manifests/install/charts/as-a-second-scheduler/templates/rbac.yaml

```diff
@@ -64,7 +64,7 @@ rules:
   verbs: ["get", "list", "watch"]
 # resources need to be updated with the scheduler plugins used
 - apiGroups: ["scheduling.sigs.k8s.io"]
-  resources: ["podgroups", "elasticquotas"]
```

Comment:
Could you please also add `elasticquotas/status`?

Reply:
Ok, done. : )